### PR TITLE
add browser build with browserify

### PR DESCRIPTION
**Pull request by Matheus-de-Souza**
I've made a version of this library to use on browsers. You just need to run `npm run build:browser` and the browserify lib generates a new a file that you can embed on web.

### CODE REVIEW

**Comment by igor-ribeiro** [comment]
Thank you! I completely forgot about the browser version.

I think a better approach would be comitting the browser version so it's on NPM, cause https://unpkg.com reads from there.

Facebook does it for React: https://unpkg.com/react/

So I would remove `dist/example.html` and would add a topic on `README.md` about importing it via `unpkg` for the browser version. And I would put the browser version inside a `umd` folder.

Can you do that?

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "mocha --reporter spec",
     "test:watch": "mocha -w --reporter min",
+    "build:browser": "node_modules/.bin/browserify -r ./index.js --standalone calendarjs -o dist/calendar.js",
```
Comment (igor-ribeiro) [inline package.json]: You can use only `browserify` here.

**Comment by igor-ribeiro** [comment]
Actually, I think we don't even need `browserify`... I can remove the `require`s statements and throw normal errors. I'll create an issue for that.

**Comment by Matheus-de-Souza** [comment]
Yeah, it's a good option, fewer dependencies. But the calendarjs will be attached to the window even for the users who use AMD, or requireJS. 

I'm implementing a custom angular calendar with your project. I liked it a lot, thanks for publishing. 😄

**Comment by igor-ribeiro** [comment]
I saw a hack like that:
```js
;(function (exports) {
  exports.calendar = //...
})}(typeof module === 'object' ? module.exports : window))
```

Maybe it works on any environment.

Thanks for the PR, and you are welcome... That was the idea of that package, being able to use it with any framework.

File: package.json (hunk 1 above)
Comment (igor-ribeiro) [reply package.json]: When you use a package on `scripts` it already looks on local `node_modules`

**Pull request closed without merging**